Lusas_Adapter/LusasAdapter.cs

```diff
@@ -151,6 +158,7 @@ public static bool IsApplicationRunning()
         //Add any comlink object as a private field here, example named:
 
         private string m_directory;
+        public double m_mergeTolerance;
```

Comment:
```suggestion
        public double m_mergeTolerance = Tolerance.Distance;
```

Reply:
Would this not change the user provided value back to `Tolerance.Distance`?

Reply:
Implemented in latest commit as discussed